open-metadata-implementation/access-services/subject-area/subject-area-spring/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/spring/SubjectAreaRelationshipRESTResource.java

```diff
@@ -1334,7 +1334,7 @@ public SubjectAreaRelationshipRESTResource() {
      *
      * @param serverName                serverName under which this request is performed, this is used in multi tenanting to identify the tenant
      * @param userId                    userId under which the request is performed
-     * @param termISATypeOFRelationship the TermISATypeOFRelationship relationship
+     * @param termIsATypeOFRelationship the TermISATypeOFRelationship relationship
```

Comment:
It is a very small point - but you may want to change the variable names to have 'Of' instead of of 'OF' for consistency

Reply:
Of course, okay